pkg/apis/kops/cluster.go

```diff
@@ -136,6 +136,12 @@ type ClusterSpec struct {
 	//   missing: default policy (currently OS security upgrades that do not require a reboot)
 	UpdatePolicy *string `json:"updatePolicy,omitempty"`
 
+	// Install Helm
+	// Valid values:
+	//   'true' Install the tiller manifest to enable helm https://github.com/kubernetes/helm
+	//   'false' Don't install it
+	EnableHelm *bool `json:"enableHelm,omitempty"`
```

Comment:
I don't know yet ...  Since it is a proper member of the k8s community, that is why I wanted it here. Also, we use it a ton. I don't want kops to be a package manager, but ... I know know.

What is a better design?  BTW we need to have it API level.